03-Bootstrap/assignment/css/_components.content.scss

```diff
@@ -1,13 +1,11 @@
 html {
-    box-sizing: border-box;
+  box-sizing: content-box;
```

Comment:
try to not do this. It's a very standard practice to use 
```
* {
box-sizing: border-box
}
```

It might seem harder nwo, but I guarantee it'll make your life easier in the future.